File: movie_merge/track_test.rb
require 'minitest/autorun'
require './movie_merge.rb'

describe Track do

  it "#parse" do
  	result = Track.parse("01:02,Track 1")
  	result.duration.must_equal SimpleTime.new(0, 1, 2)
  	result.title.must_equal "Track 1"
  end

end

File: scala_movie_merge/src/test/scala/com/eaw/moviemerge/TrackTest.scala
package com.eaw.moviemerge

import org.scalatest.FunSuite

/**
  * Created by erikwenberg on 12/25/17.
  */
class TrackTest extends FunSuite {

  test("parse") {
    val result = Track.parse("01:02,Track 1")
    assert(result.duration === SimpleTime(0, 1, 2))
  }
}
